File: android/src/test/kotlin/com/layrz/layrz_wifi/LayrzWifiPluginTest.kt
package com.layrz.layrz_wifi

import kotlin.test.Test
import kotlin.test.assertNotNull

internal class LayrzWifiPluginTest {
  @Test
  fun pluginInstantiates() {
    val plugin = LayrzWifiPlugin()
    assertNotNull(plugin)
  }
}

File: windows/test/layrz_wifi_plugin_test.cpp
#include <gtest/gtest.h>

#include "layrz_wifi_plugin.h"

namespace layrz_wifi {
namespace test {

TEST(LayrzWifiPlugin, HasDiscovery) {
  LayrzWifiPlugin plugin;
  auto result = plugin.HasDiscovery();
  EXPECT_FALSE(result.has_error());
  EXPECT_TRUE(result.value());
}

TEST(LayrzWifiPlugin, HasCurrentSsid) {
  LayrzWifiPlugin plugin;
  auto result = plugin.HasCurrentSsid();
  EXPECT_FALSE(result.has_error());
  EXPECT_TRUE(result.value());
}

}  // namespace test
}  // namespace layrz_wifi
